src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Application.vue

```diff
@@ -91,19 +91,25 @@ export default defineComponent({
           }
           break;
         default:
-          this.saveDraft();
+          this.applicationStore.saveDraft();
           this.decrementWizard();
           this.isFormValid = true;
           break;
       }
     },
+    async saveDraftAndAlertSuccess() {
+      const draftApplicationResponse = await this.applicationStore.saveDraft();
+      if (draftApplicationResponse?.applicationId) {
+        this.alertStore.setSuccessAlert("Draft application saved successfully");
+      }
```

Comment:
do we also need a failure alert here?

Reply:
We are handling failure messages/alerts in the `ApiResultHandler` class